src/lib/Behat/BrowserContext/UserPreferencesContext.php

```diff
@@ -29,4 +37,14 @@ public function iChangePassword($oldPassword, $newPassword): void
         $this->changePasswordPage->setNewPassword($newPassword);
         $this->changePasswordPage->setConfirmPassword($newPassword);
     }
+
+    /**
+     * @When I set Autosave draft option to :autosaveDraftValue value
+     */
+    public function iSetAutosaveDraftValue(string $autosaveDraftValue): void
+    {
+        $this->userSettingsPage->verifyIsLoaded();
+        $this->userSettingsPage->openAutosaveDraftEditionPage();
+        $this->userSettingsPage->setAutosaveDraftValue($autosaveDraftValue);
```

Comment:
```suggestion
        $this->userSettingsPage->disableAutosave();
```

(together with the Step rename).

I don't like that we're relying on a design details (`disabled`/`enabled` value) in our Step definition - this can change one day from a dropdown to a radio button and we'd have to refactor the Steps as well.

We can afford to have simple `disableAutosave` (and `enableAutosave`, if needed) methods.